apps/fabric-website-resources/LICENSE

```diff
@@ -12,4 +12,4 @@ The above copyright notice and this permission notice shall be included in all c
 
 THE SOFTWARE IS PROVIDED *AS IS*, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 
-Note: Usage of the fonts and icons referenced in Office UI Fabric is subject to the terms listed at https://aka.ms/fabric-assets-license
+Note: Usage of the fonts and icons referenced in Fluent UI is subject to the terms listed at https://aka.ms/fabric-assets-license
```

Comment:
Can we make a aka.ms/fluent-ui-assets-license link?

Reply:
yeah, i'm going to go through and do all that later. aka.ms/fabricdemo/fabricpen/fabricsandbox etc

Reply:
i've got a backlog item in ADO for this